src/Core/src/Platform/iOS/MauiView.cs

```diff
@@ -137,7 +137,7 @@ public override void LayoutSubviews()
 			}
 
 			CrossPlatformArrange(bounds);
-			OnLayoutChanged();
```

Comment:
Do we still need this change @kubaflo  or just the test?

Reply:
@rmarinho just the test. I forgot to remove this part. Fixed now :)